subsys/debug/etb_trace/etb_trace_private.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
```

Comment:
Explain why this file exists and perhaps the reasoning behind the prefixes (`CS`, `ATB_1` and `ATB_2`)